src/main/java/frc/robot/utils/AutoAlignment.java

```diff
@@ -22,4 +33,7 @@ public static double angleFromTarget(Alignable alignable,Pose2d currentPose){
     public static Rotation2d getAngle(Alignable alignable, double x, double y) {
         return alignable.getAngleFromDistFunc().apply(x - alignable.getX(), y - alignable.getY());
     }
+    public static void resetPid(){
```

Comment:
Move the angle calculation from the enum to here.

Reply:
done